Source/GSStoryboardLoader.m

```diff
@@ -0,0 +1,218 @@
+/* <title>GSXibLoader</title>
```

Comment:
Wrong file name and wrong attribution.

Reply:
This class will be going away.  I don't need it and the other GSStoryboard classes.